docker/compute_worker/compute_worker.py

```diff
@@ -284,8 +339,8 @@ def start(self):
         program_dir = os.path.join(self.root_dir, "program")
         ingestion_program_dir = os.path.join(self.root_dir, "ingestion_program")
 
-        self._run_program_directory(program_dir)
-        self._run_program_directory(ingestion_program_dir)
+        self._run_program_directory(program_dir, kind='program')
```

Comment:
these should be constants, like `RUN_TYPE_PROGRAM`, `RUN_TYPE_INGESTION`, what do you think?